### PR TITLE
273 update admin signup front

### PR DESCRIPTION
**Pull request by fadeomar**
#### Issues this PR relates to
#273
=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

**Review by RamyAlshurafa — CHANGES_REQUESTED** [review]

**Review by RamyAlshurafa — CHANGES_REQUESTED** [review]

File: client/src/pages/Admin/Customise/index.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+  useEffect(() => {
+    if (submitAttempt) {
+      validateForm();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [uniqueLink]);
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    setState({ submitAttempt: true });
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply client/src/pages/Admin/Customise/index.js:98]: done

File: client/src/pages/Admin/Signup/index.js [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+      }
+    } else {
+      setUser(data);
+      history.push(navRoutes.ADMIN.Complete);
+    }
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setState({ submitAttempt: true });
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply client/src/pages/Admin/Signup/index.js:110]: done

File: client/src/pages/Admin/Customise/index.js [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+            up to the platform.
+          </T.P>
+          <T.P style={{ width: '100%' }}>
+            To change your unique link edit the text in the input field below.
+          </T.P>
+        </Col>
+      </Row>
+      <Row>
+        <Col w={[4, 12, 4]} mb={7} mbM={5}>
+          <T.H3 weight="bold" mb="3">{`https://www.duetplatform.com/${
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply client/src/pages/Admin/Customise/index.js:165]: done

File: client/src/pages/Admin/Customise/index.js [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+      <Row>
+        <Col w={[4, 4, 4]}>
+          <FileUpload
+            error={fileUploadError}
+            setError={setFileUploadError}
+            setFileInfo={setUploadedFileInfo}
+            fileInfo={uploadedFileInfo}
+            uploading={fileUploading}
+            setUploading={setFileUploading}
+            category={content.fileCategories.image}
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply client/src/pages/Admin/Customise/index.js:137]: done

File: client/src/pages/Admin/Signup/index.js [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+    setState({ loading: false });
+    if (error) {
+      if (error.statusCode === 409) {
+        setState({ validationErrs: { email: error.message } });
+      } else {
+        setState({ httpError: error.message });
+      }
+    } else {
+      setUser(data);
+      history.push(navRoutes.ADMIN.Complete);
```
Comment (RamyAlshurafa) [inline client/src/pages/Admin/Signup/index.js:103]: this is undefined

File: client/src/validation/schemas/unique-link.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import { fields, createSchema, validate as _validate } from '..';
```
Comment (RamyAlshurafa) [inline client/src/validation/schemas/unique-link.js:1]: name the file same name with the page

File: client/src/validation/fields.js [hunk 7]
```diff
@@ -176,3 +177,14 @@ export const optionalRate = number().when('noDemos', {
   then: number().nullable(),
   otherwise: numberField,
 });
+
+export const organisationName = string()
+  .typeError(errMsgs.DEFAULT_REQUIRED)
+  .required(errMsgs.DEFAULT_REQUIRED);
+
+export const urlLink = string()
```
Comment (RamyAlshurafa) [inline client/src/validation/fields.js:185]: this is not link, its slug

**Review by RamyAlshurafa — CHANGES_REQUESTED** [review]
- [x] color and spacing in this section is not the same with figma
Screenshot from 2021-08-31 16-36-18
- [x] the validation doesn't work for organisation name
  Screenshot from 2021-08-31 16-38-48

File: client/src/pages/Admin/Signup/index.js (hunk 5 above)
Comment (fadeomar) [reply client/src/pages/Admin/Signup/index.js:103]: done

File: client/src/validation/schemas/unique-link.js (hunk 6 above)
Comment (fadeomar) [reply client/src/validation/schemas/unique-link.js:1]: done

File: client/src/validation/fields.js (hunk 7 above)
Comment (fadeomar) [reply client/src/validation/fields.js:185]: done

**Review by RamyAlshurafa — APPROVED** [review]

**Pull request merged**